web/src/engine/websites/decorators/CoreView.ts

```diff
@@ -0,0 +1,267 @@
+import { FetchRequest, FetchCSS } from '../../FetchProvider';
+import { type MangaScraper, type MangaPlugin, Manga, Chapter, Page } from '../../providers/MangaPlugin';
+import type { Priority } from '../../taskpool/TaskPool';
+import * as Common from './Common';
+
+export const mangaPaths = ['/series', '/series/oneshot', '/series/finished'];
+export const queryMangaTitleFromURI = '.series-header-title';
+
+// The query to select *all* required data for *one* manga
+export const queryMangas = [
```

Comment:
There are so many queries, it may be better when each website provides the query instead of adding all of them to the decorator